sdcm/cluster.py

```diff
@@ -2894,7 +2894,7 @@ def node_setup(_node):
                                             args=(node,))
             setup_thread.daemon = True
             setup_thread.start()
-            time.sleep(30)
+            setup_thread.join(timeout=300)
```

Comment:
After this change, the node_setup won't execute in parallel.
It's good to avoid bootstrapping issue.

If the node_setup doesn't finish, will it be interrupted unexpectedly?

wait_for_init() will also be used in decommission nemesis, it will be a little bit slow to bootstrap the new node with heavy load in longevity jobs.  300 seconds might be not enough.

Reply:
As Amos commented, your solution will break the parallelism of the logic here.
Per Shlomi's comment, let's increase the timeout to 2 minutes, but we need probably to have a conf parameter since in scale test of 500 nodes we will need to wait 1000 minutes (~16.6 hours) just to bootstrap a cluster...
@roydahan @dkropachev WUT?

Reply:
This is not a good solution.
@amoskong solution to increase the sleep is better for now, but still not ideal (#2314 ).